### ISSUE TITLE
BLE signal improvement

### ISSUE THREAD

**Issue by yunnanpl** [yunnanpl/esp32_python_eq3#22]
The common for the whole EQ3 control is weak BLE signal. Usually weak means lower than -90. Such devices tend to disappear, or loss connection. Such connection needs to be retried multiple times, making the ESP32 less responsive.
I did tried multiple solution with soldering antennas, but I am just not good at it, and it tends to break.
Signal of -80 to -86 is completely fine and stable, and even stronger signal is more than needed.

Lately I found this:
https://peterneufeld.wordpress.com/2021/10/14/esp32-range-extender-antenna-modification/
An inductive range extender, which is just glued (or taped in my case). It requires no soldering, it can be removed and remade any time as no permanent change is "inflicted".
This trick improves the BLE signal by 5 to 10 dBm units which really does wonders, and is very easy to apply in few minutes with a piece of wire and tape.

TLDR: Take a 58 mm piece of insulated wire. Bend it in such a way that part of it lies on the ESP32 antenna. The rest should extend outside of ESP32. Fix it with tape (temporary) or glue (more permanent). Done, your signal should go up 5 to 10 dBm !

**Comment by Saiuday9787** [comment]
Hiii, I am trying for LONG RANGE BLE (>300meters) using ESP32 C3 mini-1.

I tried esp_ble_tx_power_set(ESP_BLE_PWR_TYPE_ADV, ESP_PWR_LVL_P21);  at server side and client side, still i am able to get Line of site upto 100meters and able to tx via one floor concrete slab. still my requirement is not satisfied, 
Can you suggest on how to achieve above 300meters range.
